### ISSUE TITLE
Write links and summaries of each notebook in the README

### ISSUE THREAD

**Issue by v0dro** [SciRuby/sciruby-notebooks#1]
Would make it easier to browse through the collection.

**Comment by minad** [comment]
Yes, however the names should also be chosen well, since they will appear in the notebook browser.

Maybe the jupyter filebrowser could help here? Let's ask them https://github.com/jupyter/notebook/issues/130

**Comment by v0dro** [comment]
Well we can do it all manually for now :P
